### ISSUE TITLE
bad_bzdata_permissions

### ISSUE THREAD

**Issue by endotronic** [tom300z/backblaze-personal-wine#9]
Have any of you seen and resolved this warning? Backblaze seems to back up just fine for me, but this notification pops up consistently.
![bad_bzdata_perms](https://user-images.githubusercontent.com/1331112/155897590-122aa8b7-d864-4b65-ab97-63f1fee67db6.png)

I have (recursively) set permissions on /wine/drive_c/ProgramData/Backblaze/bzdata to 755 and 777 and neither configuration seems to resolve this issue. Maybe this is just an artifact of emulating Windows via Wine.

**Comment by endotronic** [comment]
I should add that clicking "Learn More" dismisses the dialog without any helpful result. I assume it tries to launch a browser and fails. The dialog reappears in a few minutes after it is dismissed.

**Comment by ad3m3r5** [comment]
@endotronic, did you ever find a solution to this? I'm coming across the same issue, but it seems to be very inconsistent.

**Comment by endotronic** [comment]
I haven't yet. I plan to look through their logs for clues but haven't found the time. I have another issue as well that I'll be looking for. It's unrelated to this one, but I am finding that it keeps sending my large files (several GB in size) over and over while backing up in continuous mode.

**Comment by semool** [comment]
This is wrong mounting. The Client cant create its Directory for Chunk Uploads.
Your first mount must go directly to data. It must big enough to chunk the largest File you will Backup. A empty Folder.
-v /mnt/backblaze-temp:/data    <-- this is drive D:

All the other dirs you want to Backup goes in Data(D:) Subdirs .
-v /mnt/backup1:/data/backup1
-v /mnt/backup2:/data/backup2
